src/main/resources/validationrules/civil/civil-admin-mapping.drl

```diff
@@ -234,11 +234,13 @@ end;
 
 /*
  * CIVIL admin "hearing-centre-team-leader" Org role mapping. v1.6
+ * Made obsolete in COT-906 - disabled by CIVIL_WA_2_1 flag.
```

Comment:
this rule will need to be removed some time after `CIVIL_WA_2_1` is enabled, tested & signed off. i think you should raise a new ticket and add comment `To be removed in COT-<ticket-num>`, see rule `v11_civil_hearing_centre_admin_org_role` for example.

Reply:
I have created ticket DTSAM-591 to handle the deletion of rules made obsolete by COT-906